src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Languages/en.xaml

```diff
@@ -4,5 +4,6 @@
     
     <system:String x:Key="Microsoft_plugin_indexer_copy_path">Copy path (Ctrl+C)</system:String>
     <system:String x:Key="Microsoft_plugin_indexer_open_containing_folder">Open containing folder (Ctrl+Shift+E)</system:String>
-
+    <system:String x:Key="Microsoft_plugin_indexer_run_as_administrator">Run As Administrator (Ctrl+Shift+Enter)</system:String>
```

Comment:
Please change to "Run as Administrator" to be consistent withe the other tooltips.
This mistake exist in all language files.

Reply:
It is with a capital 'A' in 'As' like in the other tooltips.

Program plugin  - https://github.com/microsoft/PowerToys/blob/master/src/modules/launcher/Plugins/Microsoft.Plugin.Program/Languages/en.xaml#L35
Shell Plugin - https://github.com/microsoft/PowerToys/blob/master/src/modules/launcher/Plugins/Microsoft.Plugin.Shell/Languages/en.xaml#L13. 

Can you pls clarify if this is what you are referring to? If not, can you provide more details.

Reply:
What I mean is we write for the other buttons "Open **c**ontaining folder" and "Copy **p**ath". But for the admin button we write "Open **A**s Administrator".

Reply:
Created an issue to track it - #4842. We can make that change as another PR.